3000/gentx/funky-gentx/funky.txt

```diff
@@ -0,0 +1 @@
+kava1gvv0l5se2zmgn6tra20f63u2rmjv8lpge7p25y
```

Comment:
Please use an account that is different than the delegator address in your gentx. Thanks.

Reply:
updated :) sorry for the sloppiness!